be/src/exec/vectorized/connector_scan_node.cpp

```diff
@@ -503,6 +509,13 @@ Status ConnectorScanNode::close(RuntimeState* state) {
 
 Status ConnectorScanNode::set_scan_ranges(const std::vector<TScanRangeParams>& scan_ranges) {
     _scan_ranges = scan_ranges;
+    if (scan_ranges.size() == 0) {
```

Comment:
To handle empty scan ranges case, here I add a placehold scan range, to force data source provider to create at least one data source.

Reply:
why not insert it in `FE`, because scan range provided by specific connector on `FE`

Reply:
> why not insert it in `FE`, because scan range provided by specific connector on `FE`

It's ok to change in FE. But from FE side, it makes non-sene, because to JDBC scanner there is no scan range.

Reply:
It's ok to change in FE & BE. Here I just want to keep it simple and don't make any change on FE.